docs/channels.md

```diff
@@ -0,0 +1,103 @@
+# Channel Guide
+
+The Fabric Discord server (Fabricord) is a relatively large community, containing a wide variety of channels — mostly relating directly to Fabric, but also including a lively off-topic community. This document serves as a run-down of all of our channels, and how they're intended to be used.
+
+We also include channel descriptions directly on Discord — please read them.
+
+## Information Category
+
+This category contains informational channels, providing important information on the Fabric project and its community, as well as information on Fabric and Minecraft updates and official communications from moderators. Users may not send messages in these channels — they're read-only.
+
+* `#tiny-potato`: 
+  
+  ![](https://cdn.discordapp.com/attachments/552976908070027270/552978960896753665/2015-04-26_11.13.01.png)
+
+* `#welcome`: The primary Discord information channel and the one you should be familiar with. This channel contains information on how the community is run, how we expect you to behave, and what you can do if there's a problem.
+
+* `#announcements`: New Minecraft releases and important Fabric news is announced in this channel.
+
+* `#live-update-updates`: Information on the Fabric update process and how it adjusts to Minecraft updates is posted here, along with information we dig up during that process.
+
+* `#moderator-messages`: Important communications from the moderation team, directed at the community as a whole. We suggest you avoid muting this channel, for the sake of staying up to date on community policies and not being caught out by changes.
+
+## Showcase Category
+
+This category is where you go to show off your Fabric-related projects. Please note that this category has only one discussion channel — the others should not be used for conversation.
+
+All projects posted in these channels need to be related to Fabric somehow — for example: mods and libraries, or services for Fabric users.
+
+* `#new-releases`: You can post about the initial public release of your new projects here. We'll also allow posts for the first public release of a completely reworked project.
+
+* `#update-releases`: You can post when you release an update to a project that's already been released. Do not post newly-released projects here.
+
+* `#showcase`: This channel can be used to showcase the things you're working on - for example, a new feature you've added to a mod. As many members of our community are not programmers, we ask that you post in-game screenshots and videos where possible instead of code — although we do recognise that some projects can only be showcased through code.
+
+* `#showcase-discussion`: Use this channel to discuss anything that's been posted in any of the other three channels. Please note that **this is not an off-topic channel** — we have more than one of those in the community category if you wish to talk about something else.
```

Comment:
Should this channel name be changed to reflect that it is a discussion for all 3 above channels, rather than just showcase?

Reply:
That might be an idea, we're still talking about channel structure. That'd be out of scope of this PR, though — we can take a look at it later.